openshift/README.md

```diff
@@ -74,8 +71,7 @@ oc create -n k8vopl-<env> secret generic getok-sc-ches-secret \
 ```
 
 ```sh
-oc create -n k8vopl-<env> secret generic getok-sc-keycloakint-secret \
-  --type=kubernetes.io/basic-auth \
+oc create -n k8vopl-<env> secret generic getok-sc-keycloakdev-secret type=kubernetes.io/basic-auth \
```

Comment:
I think we should have this be `getok-sc-keycloak-dev-secret` instead (and updated for everything else referencing it in order to be consistent with all the other secret nomenclatures.

Reply:
Can we do this with next PR, might not want to clash with other PRs using the configs.

Reply:
Ok. I can wrap this as a part of the upcoming db seed changes as that will do some pipeline modifications anyways.